ignition_ros2_control/src/ignition_system.cpp

```diff
@@ -320,13 +320,28 @@ void IgnitionSystem::registerSensors(
     });
 }
 
-hardware_interface::return_type
-IgnitionSystem::configure(const hardware_interface::HardwareInfo & actuator_info)
+CallbackReturn
+IgnitionSystem::on_init(const hardware_interface::HardwareInfo & actuator_info)
 {
-  if (configure_default(actuator_info) != hardware_interface::return_type::OK) {
-    return hardware_interface::return_type::ERROR;
+  RCLCPP_WARN(this->nh_->get_logger(), "On init...");
+  if (hardware_interface::SystemInterface::on_init(actuator_info) != CallbackReturn::SUCCESS)
+  {
+    return CallbackReturn::ERROR;
   }
-  return hardware_interface::return_type::OK;
+  return CallbackReturn::SUCCESS;
+}
+
+CallbackReturn IgnitionSystem::on_configure(
+  const rclcpp_lifecycle::State & /*previous_state*/)
+{
+  RCLCPP_INFO(
+    this->nh_->get_logger(), "Configuring ...please wait...");
+
```

Comment:
```suggestion
```

Reply:
Pushed this modification, sorry for the many pushes. I have yet to add an automatic signature to my commits

Reply:
No worries, I will squash when it will be ready to merge